huaweicloud/services/acceptance/apig/resource_huaweicloud_apig_group_test.go

```diff
@@ -46,40 +52,102 @@ func TestAccGroup_basic(t *testing.T) {
+				ImportStateIdFunc: testAccGroupImportStateFunc(rName),
+			},
+			{
+				Config: testAccGroup_basic_step4(baseConfig, name),
+				Check: resource.ComposeTestCheckFunc(
+					rcWithVariables.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rNameWithVariables, "environment.#", "2"),
+				),
+			},
+			{
+				Config: testAccGroup_basic_step5(baseConfig, name),
+				Check: resource.ComposeTestCheckFunc(
+					rcWithVariables.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rNameWithVariables, "environment.#", "2"),
+				),
+			},
+			{
+				Config: testAccGroup_basic_step6(baseConfig, name),
+				Check: resource.ComposeTestCheckFunc(
+					rcWithUrlDomain.CheckResourceExists(),
+					// since the order in the schema is inconsistent with the order of data obtained by the API, other parameters are not verified.
+					resource.TestCheckResourceAttr(rNameWithUrlDomain, "url_domains.#", "2"),
+					resource.TestCheckResourceAttrSet(rNameWithUrlDomain, "url_domains.0.min_ssl_version"),
+					resource.TestCheckResourceAttr(rNameWithUrlDomain, "url_domains.0.is_http_redirect_to_https", "false"),
+				),
+			},
+			{
+				Config: testAccGroup_basic_step7(baseConfig, name),
+				Check: resource.ComposeTestCheckFunc(
+					rcWithUrlDomain.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rNameWithUrlDomain, "url_domains.#", "1"),
+					resource.TestCheckResourceAttr(rNameWithUrlDomain, "url_domains.0.name", "www.terraform.test3.com"),
+					resource.TestCheckResourceAttr(rNameWithUrlDomain, "url_domains.0.min_ssl_version", "TLSv1.1"),
+					resource.TestCheckResourceAttr(rNameWithUrlDomain, "url_domains.0.is_http_redirect_to_https", "true"),
+				),
+			},
+			{
+				// Check whether illegal URL domain ​​can be intercepted normally (update phase).
+				Config:      testAccGroup_basic_step8(baseConfig, name),
+				ExpectError: regexp.MustCompile("error binding domain name to the API group"),
+			},
+			{
+				Config: testAccGroup_basic_step9(baseConfig, name),
+				Check: resource.ComposeTestCheckFunc(
+					rcWithDomainAccess.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rNameWithDomainAccess, "domain_access_enabled", "false"),
+				),
+			},
+			{
+				Config: testAccGroup_basic_step10(baseConfig, name),
+				Check: resource.ComposeTestCheckFunc(
+					rcWithDomainAccess.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rNameWithDomainAccess, "domain_access_enabled", "true"),
+				),
 			},
 		},
 	})
 }
 
-func testAccGroupImportStateFunc() resource.ImportStateIdFunc {
+func testAccGroupImportStateFunc(rsName string) resource.ImportStateIdFunc {
 	return func(s *terraform.State) (string, error) {
-		rName := "huaweicloud_apig_group.test"
-		rs, ok := s.RootModule().Resources[rName]
+		rs, ok := s.RootModule().Resources[rsName]
 		if !ok {
-			return "", fmt.Errorf("resource (%s) not found: %s", rName, rs)
+			return "", fmt.Errorf("resource (%s) not found: %s", rsName, rs)
```

Comment:
Actually, the variable "rs" is zero here.
It is unnecessary to print it.

Reply:
sure